.github/workflows/conformance-k8s-kind.yaml

```diff
@@ -26,6 +26,7 @@ env:
   # renovate: datasource=github-releases depName=cilium/cilium-cli
   cilium_cli_version: v0.14.5
   cilium_cli_ci_version:
+  CILIUM_CLI_MODE: helm
```

Comment:
I actually try with global env like this, but i faced a couple of issue like flag abc is not supported for cilium cli command (other than cilium install), not sure if it works here, and for all cases.

Reply:
yeah let's see if i'm lucky enough.